### DISCUSSION TITLE
Multiple model download

### DISCUSSION THREAD

**Discussion by cdefghijkl** [TheLastBen/fast-stable-diffusion#1335] [Ideas]
![Screenshot 2023-01-10 at 20-59-20 Google Colaboratory](https://user-images.githubusercontent.com/121331850/211570761-1d5177f6-130a-4f7d-945f-a55517c71331.png)

Can you add one or two line below model_link to download multiple models at once? For merging models purpose. So far I have to download it all of them to my drive folder first and it's tiring.

Or like this colab which can download multiple models from civitai and store it to colab temporary storage.
https://colab.research.google.com/drive/1aQ5nXTfLWHhZi7GOfXteLKg5OT1X-aBZ

**Comment by TheLastBen** [comment]
I work on that later, meanwhile, use 
```
%cd /content
!wget <link to the model>
```
to download the model to the temporary storage

**Reply by cdefghijkl** [reply]
Thank you.
